src/modules/site-v2/base/utils/local_file.py

```diff
@@ -83,6 +84,9 @@ def remove(self):
       return True
     except FileNotFoundError:
       return False
+    except Exception as e:
```

Comment:
We might need to expand some of the exceptions, e.g.: file permissions/file locks/etc, before returning `True`.

Reply:
sounds good.